src/plugins/WebOfTrust/l10n/lang_de.l10n

```diff
@@ -96,7 +96,6 @@ KnownIdentitiesPage.FiltersAndSorting.Header=Filter und Sortierung
 KnownIdentitiesPage.FiltersAndSorting.ShowOnlyNicksContaining=Nur Namen anzeigen, die folgendes beinhalten
 KnownIdentitiesPage.FiltersAndSorting.SortIdentitiesBy.Ascending=Aufsteigend
 KnownIdentitiesPage.FiltersAndSorting.SortIdentitiesBy.Descending=Absteigend
-KnownIdentitiesPage.FiltersAndSorting.SortIdentitiesBy.Edition=Edition
```

Comment:
Why is this string removed? The code still uses it, It still shows on the Transifex web interface, and cannot have been modified since it is marked as reviewed.
Bug in your scripts?

Notice: Please keep this pull request open, I've already invested too much time in getting it merged. I'll fix it manually for this string, and other languages will get fixed with the next one. Also, before you do another pull request to fix l10n, please first wait until I tell you to update the English source strings. There are scheduled modifications to the English l10n which I might do soon.